lib/tiddle/model_name.rb

```diff
@@ -1,11 +1,17 @@
 module Tiddle
```

Comment:
can we just leave a big comment on this class clarifying this is a monkey patch of a gem class, and describe the rationale there.

otherwise looks good 👍

Reply:
oh, i didn't realize this was a fork